docs/topics/multiplatform/multiplatform-hierarchy.md

```diff
@@ -50,6 +53,25 @@ kotlin {
 }
 ```
 
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
+kotlin {
+  // Enable the default target hierarchy:
+    targetHierarchy.default {
```

Comment:
It feels that `targetHierarchy.default()` is better in this case because `{   }` implicitly suggests adding something inside.
But IDE reports `GroovyImplicitNullArgumentCall` for round brackets 😒